# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: dom/cookiestore/PCookieStore.ipdl
# Commit: 2081d42e6ccd
# Full Hash: 2081d42e6ccdcc39b3d1b2e6523a8d44fed85390
# Author: Andrea Marchesini <amarchesini@mozilla.com>
# Date: 2024-09-13 04:28:43
# Regressor Bug: 1475599
# File Overlap Count: 1
# Description:
#   Bug 1475599 - part 8 - CookieStore API - e10s support, r=smaug,cookie-reviewers,timhuang
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D217574
# ==============================================================================

diff -r c6e20b939a21 -r 2081d42e6ccd dom/cookiestore/PCookieStore.ipdl
--- a/dom/cookiestore/PCookieStore.ipdl	Thu Sep 12 16:48:21 2024 +0000
+++ b/dom/cookiestore/PCookieStore.ipdl	Thu Sep 12 16:48:21 2024 +0000
@@ -6,6 +6,7 @@
 
 using mozilla::OriginAttributes from "mozilla/ipc/BackgroundUtils.h";
 using struct mozilla::void_t from "ipc/IPCMessageUtils.h";
+using struct nsID from "nsID.h";
 
 namespace mozilla {
 namespace dom {
@@ -44,13 +45,15 @@
                    int64_t expires,
                    nsString path,
                    int32_t sameSite,
-                   bool partitioned) returns (bool v);
+                   bool partitioned,
+                   nsID operationId) returns (bool waitForNotification);
 
   async DeleteRequest(nsString domain,
                       OriginAttributes attrs,
                       nsString name,
                       nsString path,
-                      bool partitioned) returns (bool v);
+                      bool partitioned,
+                      nsID operationID) returns (bool waitForNotification);
 
   async Close();
 